doc/team_practices.md

```diff
@@ -0,0 +1,37 @@
+# C2 Team Practices
+
+This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+
+## Workflow
+
+All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
+
+Stories are clarified and prioritized in weekly iteration planning meetings (IPM). Team members self-select story ownership.
+
+Here is an example workflow:
+
+* Click **Start** on the story in the story tracker.
+
+* Create a new Git branch from master. The naming convention is *storyId*-*short-description*. Example:
+```
+% git checkout master
+% git checkout -b 123456-fix-timezones
+```
+
+* Write your code and tests and documentation.
```

Comment:
Maybe link to [how to run tests](https://github.com/18F/C2/blob/master/doc/setup.md#running-tests)?